### ISSUE TITLE
Inconsistent layout in settings -> Metadata

### ISSUE THREAD

**Issue by jacogr** [polkadot-js/apps#4394]
Additionally, it also has this -

![image](https://user-images.githubusercontent.com/1424473/104461917-3ebb4f00-55b0-11eb-9ab4-44b5fcdaf2d5.png)

**Comment by jhonalino** [comment]
![Screen Shot 2021-01-24 at 1 07 11 PM](https://user-images.githubusercontent.com/13143458/105643503-1c91be80-5e45-11eb-850c-a8b8d79b2a55.png)
this is currently what I see right now sir. what look are we going for here sir jacogr?

**Comment by jacogr** [comment]
Aha. My little pet peeve :) So the issue for me is that is looks completely different from everything else. 

So it need basically the same header box layout as the rest. So basically needs to be boxed and sectioned. (I think all the rest are now tables, so it may not translate easily but the styling indeed would)

TL;DR needs to be wrapped in a box with the header - to look like what we have everywhere. 

The button is just that it acts weird, one again not consistent with the rest of the UI. It has some weird styling in there.

**Comment by jhonalino** [comment]
![Screen Shot 2021-01-24 at 5 43 59 PM](https://user-images.githubusercontent.com/13143458/105651442-c5ebab00-5e6b-11eb-9332-4f78021293be.png)
![Screen Shot 2021-01-24 at 5 43 37 PM](https://user-images.githubusercontent.com/13143458/105651448-cb48f580-5e6b-11eb-94dd-c92cfda6c325.png)

how do we look, sir? please do let me know if it looks good or if I should change anything! thank you sir!

**Comment by jhonalino** [comment]
also I need to test out the extensions. how do I have it so that extensions table is populated?

**Comment by jacogr** [comment]
For the extension test connect to something non-Polkadot, eg. Westend would work. That should prompt for an update.

**Comment by jacogr** [comment]
It think it looks good. We may be able to try pit the QR in the second one as well. Put on a last column with a rowSpan of 6 with the width of the second area as set. May work. 

If not the above is already 100x better.

**Comment by jhonalino** [comment]
![Screen Shot 2021-01-25 at 4 23 00 AM](https://user-images.githubusercontent.com/13143458/105705881-a97c5c80-5ec5-11eb-9533-8757508d0560.png)
![Screen Shot 2021-01-25 at 4 22 35 AM](https://user-images.githubusercontent.com/13143458/105705885-aa14f300-5ec5-11eb-843e-e787585a7391.png)
good idea! the rowspan looks good! lets go with this?

**Comment by jacogr** [comment]
All done. Thank you, once again.
